base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
```

Comment:
I spent a couple of hours trying that, and it is turning out to be a mess.
I'm facing a problem as to how to call the invoke method of the dialog box Operator class, from the invoke method of the `MBDynReadLog` class.